LEAF_Request_Portal/utils/db_procedures/procedure_ReplaceErrorMsg.sql

```diff
@@ -0,0 +1,54 @@
+/**************************************************************************************
+CREATED BY : Liping Wang
+CREATION DATE : 06/08/2020
+DESCRIPTION: Stored procedure to search Database Errors from a specific column on a specific table, to replace the DB Errors with a emply string.
+It accepts 'find' string and 'replce' as parameters to replace 'find' by 'replce' on the column in entire database.
+**************************************************************************************/*/
+
+/*
+USE [LEAF_Database]
+GO
+*/
+
+/********* Create procedure for replacing error string ****************/
+CREATE DEFINER=`tester`@`%` PROCEDURE `ReplaceErrorMsg`(find varchar(255), 
```

Comment:
might want to consider making the user dynamic or adding instructions to change the user depending on db setup